helm-chart/binderhub/templates/dind/daemonset.yaml

```diff
@@ -41,6 +41,9 @@ spec:
           - dockerd
           - --storage-driver={{ .Values.dind.storageDriver }}
           - -H unix://{{ .Values.dind.hostSocketDir }}/docker.sock
+          {{- if .Values.dind.daemonset.extraArgs }}
+{{ toYaml .Values.dind.daemonset.extraArgs | indent 10 }}
```

Comment:
Should the indent by 10 or 12? Or does it depend on if we use `{{- if .Values.dind.daemonset.extraArgs }}` vs `{{ if .Values.dind.daemonset.extraArgs -}}` (used above on L31). I can't hold all the templating rules in my head :-( maybe the easiest is to run the chart and double check how it is rendered.

Reply:
I was also struggling to understand what's the right indent, but I did run the chart to check:

```
helm template \
   --set jupyterhub.proxy.secretToken="$(openssl rand -hex 32)"  \
   --set jupyterhub.hub.services.binder.apiToken=$(openssl rand -hex 32) \
   --set dind.enabled=true \
   --set 'dind.daemonset.extraArgs={--mtu=1234,--arg2,--arg3=foobar}'  \
   binderhub  | grep -A 5 dockerd
          - dockerd
          - --storage-driver=overlay2
          - -H unix:///var/run/dind/docker.sock
          - --mtu=1234
          - --arg2
          - --arg3=foobar
```

Reply:
btw, if using the `{{ if -}}` it will break the indent as the first line will already be indented so it will show up as:

```
                     - --mtu=1234
          - --arg2
```

Reply:
Cool! And thanks for double checking. Templating languages seem to be non intuitive in 90% of cases ;)